src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothEventArgs.cs

```diff
@@ -628,6 +668,46 @@ public BluetoothAudioProfileType ProfileType
         }
     }
 
+    /// <summary>
+    /// An extended EventArgs class contains the connection state and the address of the remote Bluetooth device.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class AgScoStateChangedEventArgs : EventArgs
+    {
+        int _result;
+        private bool _isOpened;
+
+        internal AgScoStateChangedEventArgs(int result, bool isOpened)
+        {
+            _result = result;
+            _isOpened = isOpened;
+        }
+
+        /// <summary>
+        /// The result.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int Result
```

Comment:
`Result` must have a preceding request.
Result is not proper in EventArgs

Reply:
I will remove the 'Result'.